training/04_editing_eml.Rmd

```diff
@@ -87,3 +87,22 @@ qa_package(mn, resource_map_pid)
 * Then write, validate, and update your package.
 * Use the <a href = 'https://nceas.github.io/datateam-training/reference/final-checklist-before-notifying-submitter.html' target='_blank'>checklist</a> to review your submission.
 * Make edits where necessary, and publish updates as needed.
+
+
+## General tips for dealing with data packages
+
+In this section, we'll be discussing the general guidelines when dealing with data packages - specifically, when to touch a PI's data, what constitutes as a robust and comprehensive methods/abstract section, and when it is necessary to followup with PIs about dataset and metadata inquiries.
+
+One of the biggest dilemmas when handling data packages is deciding when it's appropriate to touch a PI's data. As a rule of thumb, data entries themselves should always be fixed. Our goal is to clean-up the files, not to revise the data. 
+
+Data file conversions are generally alright. Conversions are needed when the existing file format may become unavailable in the future. In the case of Excel sheets, since they are stored in a proprietary software, the files may eventually become inaccessible. Storing them in a non-proprietary software, such as a csv or text file, prevents it from becoming extinct. 
+
+Data files should also be revised if there are formatting issues. For instance, column headers should be fixed if there exists special characters, spaces, etc. Similarly, if there exists blank entries without missing value codes, they should be filled in with `NA` values with their respective explanations. These changes should be documented somewhere in the description or in an attribute definition.
```

Comment:
I'm not totally sure we want to be doing this.  @jeanetteclark what do you think?

Reply:
The first paragraph of this is correct, but not the second. I don't think we should be reformatting plain text files unless there is an extremely compelling reason. This is a bit of a diversion from some of our previous workflow policies but I've talked with Matt and Dominic about this a fair bit